pyomo/contrib/simplification/build.py

```diff
@@ -0,0 +1,95 @@
+from pybind11.setup_helpers import Pybind11Extension, build_ext
+from pyomo.common.fileutils import this_file_dir, find_library
+import os
+from distutils.dist import Distribution
+import sys
+import shutil
+import glob
+import tempfile
+from pyomo.common.envvar import PYOMO_CONFIG_DIR
```

Comment:
Not super important, but the ordering on this is hurting my brain. Can you put the system-level imports (`os`, `shutil`, etc.) at the top and the pyomo ones at the bottom?